IdentityCore/src/MSIDConstants.m

```diff
@@ -61,3 +61,5 @@
 
 NSString *const MSID_DEFAULT_AAD_AUTHORITY       = @"https://login.microsoftonline.com/common";
 NSString *const MSID_DEFAULT_MSA_TENANTID        = @"9188040d-6c67-4c5b-b112-36a304b66dad";
+
+NSString *const MSID_ACL_OWNER_AUTHORIZATION_TAG = @"ACLAuthorizationChangeACL";
```

Comment:
nit: I'd keep this inside MSIDMacKeychainTokenCache

Reply:
Addressed.